### ISSUE TITLE
Feature idea: Orcid.org support

### ISSUE THREAD

**Issue by MaxVonMustermann** [StevenGLee/zotero-author-browser#20]
Hey there, thank you for the great plugin. This is what I was looking for! It would be cool, if you added "Search author on orcid.org"-button to the context menu. There is already the possibility to search with Google Scholar, but from my point of view, orcid.org would be a nice addition.
So that pressing on the button basically inserts the author name into https://orcid.org/orcid-search/search?searchQuery=NAME so the author can be searched on the platform.
It would be highly appreciated. Let me know what you think. Thank you very much in advance!

**Comment by StevenGLee** [comment]
Good idea, but I'm really busy recently. Would you like to implement it and make a pull request?
